protocols/manage/propagate.go

```diff
@@ -173,7 +179,7 @@ func (p *Propagate) RegisterOnDone(fn func(int)) {
 // RegisterOnData takes a function that will be called once all connections
 // are set up. The argument to the function is the number of children that
 // sent OK after the propagation
```

Comment:
I know it's not from this PR but I don't get the comment with respect to the function name + implementation. Is not the given function will be called when the protocol receives the data ?